packages/collector/dummy-app/package-lock.json

```diff
@@ -5,36 +5,36 @@
   "requires": true,
   "dependencies": {
     "@instana/autoprofile": {
-      "version": "1.100.1",
-      "resolved": "https://registry.npmjs.org/@instana/autoprofile/-/autoprofile-1.100.1.tgz",
-      "integrity": "sha512-IvCQsyUyN0nkfHbVwEYTtXw42+9DEcK72lLmHN7womrp33ehRnRevCWE/MuJxNaPgnTVbHpeDzVexGfbPw+MiQ==",
+      "version": "1.110.5",
```

Comment:
The changes in this file are unrelated to the actual change. I just updated the Instana package to latest for the test application.